src/modelio.cs

```diff
@@ -2971,6 +2982,10 @@ interface MDLTransformStack : NSCopying, MDLTransformComponent {
 		[Export ("addMatrixOp:inverse:")]
 		MDLTransformMatrixOp AddMatrixOp (string animatedValueName, bool inverse);
 
+		[iOS (13,0), Mac (10,15), TV (13,0)]
+		[Export ("addOrientOp:inverse:")]
+		MDLTransformMatrixOp AddOrientOp (string animatedValueName, bool inverse);
+
```

Comment:
should not this return a `MDLTransformOrientOp` ?https://developer.apple.com/documentation/modelio/mdltransformstack/2890002-addorientop?language=objc